### DISCUSSION TITLE
having broken finger issue

### DISCUSSION THREAD

**Discussion by xalteropsx** [huggingface/diffusers#8357] [General]
i have most time extra finger and broken finger and deformed finger issue like 80% time  @tolgacangoz  is their any suggestion u can how can i counter it is their any model design for it ?

**Comment by tolgacangoz** [comment]
This is one of the biggest problems in this field. [Could you try these negative prompts?](https://huggingface.co/spaces/stabilityai/stable-diffusion/discussions/7857)

**Reply by xalteropsx** [reply]
sorry just woke yah something like this

BadDream,skewed eye, weird hands, broken fingers, extra fingers, missing fingers, mutated hands,extra hands,extra arms

its really funny only finger destroy whole image  most time negative prompt act little weird as well like something it alter good prompt result and make more worse

**Reply by xalteropsx** [reply]
![9](https://github.com/huggingface/diffusers/assets/103671642/11c3a46e-aa5f-4159-9b02-c13dac62c670)
![9](https://github.com/huggingface/diffusers/assets/103671642/fa87e2c0-3e8f-4d2c-8a80-28a1bcb9ca6b)
![0](https://github.com/huggingface/diffusers/assets/103671642/2e432060-a92a-4fa6-9f73-763359997223)
![0](https://github.com/huggingface/diffusers/assets/103671642/a92e6aa4-53b9-44d9-80ae-299d9d99ef2f)

**Reply by tolgacangoz** [reply]
[There is even a bounty for this problem currently.](https://x.com/levelsio/status/1792970963421638820)

**Reply by xalteropsx** [reply]
> [There is even a bounty for this problem currently.](https://x.com/levelsio/status/1792970963421638820)

lol someone even place bid xD i wish i can make the model

**Reply by tolgacangoz** [reply]
[😂](https://x.com/AntonObukhov1/status/1796534912738267205)

**Reply by tolgacangoz** [reply]
FYI: [Reddit page: One of the best uses for multi-controlnet ( from @toyxyz3 )](https://www.reddit.com/r/StableDiffusion/comments/11c4m4q/one_of_the_best_uses_for_multicontrolnet_from/)

**Reply by xalteropsx** [reply]
look bit nice approach but like hirafix more >.</ it done good job also my bad on negative prompt mistake previously :< but all good
